src/authorizations/components/redesigned/ResourceAccordion.tsx

```diff
@@ -47,25 +56,52 @@ class ResourceAccordion extends Component<OwnProps> {
 
   getAccordionBody = (resourceName, resource) => {
     const {permissions, onIndividualToggle} = this.props
+    const permissionNames = []
     if (resourceName === 'Telegrafs') {
+      for (const [key, value] of Object.entries(
```

Comment:
i think you want to do something like:

```ts
for (const [_, value] of Object.entries(
```
since we're not using `key`

Reply:
ah! that's exactly why its failing.. thanks 😊